### PR TITLE
Updated the Gardener API documentation

### PR DESCRIPTION
**Pull request by n-boshnakov**
**What this PR does / why we need it**:
This PR:
- updates the screenshots in Working with Projects
- updates the documentation on Accessing the Gardener API
- fixes broken links

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```doc developer
NONE
```

### CODE REVIEW

File: docs/usage/gardener-api.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+   ![Enter service account details](../images/02-enter-service-account-details.png)
+
+   The following *Roles* are available:
+
+   | Role | Granted Permissions |
+   |:---|:---|
+   | *Admin* | Fully manage resources inside the project, except for member management. Also the delete/modify permissions for `ServiceAccount`s are now deprecated for this role and will be removed in a future version of Gardener, use the `Service Account Manager` role instead. |
+   | *Viewer* | Read all resources inside the project except secrets. |
+   | *UAM* | Manage human users or groups in the project member list. Service accounts can only be managed admins. |
+   | *[Service Account Manager](https://github.com/gardener/gardener/blob/master/docs/usage/service-account-manager.md)* | This allows to fully manage service accounts inside the project namespace and request tokens for them. For security reasons this role should not be assigned to service accounts, especially since a service account should not be able to refresh tokens for itself. |
```
Comment (petersutter) [inline docs/usage/gardener-api.md:39]: Meanwhile we have updated the texts. This should be aligned with https://github.com/gardener/dashboard/blob/4cdb3995be4b6049e3565abdc26f3837bb9dbbf9/frontend/src/components/dialogs/MemberHelpDialog.vue#L34-L46

File: docs/usage/gardener-api.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+title: Accessing the Gardener API
+---
+
+# Accessing the Gardener API Through the Dashboard
+
+## Overview
+
+The cluster operations that are performed manually in the dashboard or via `kubectl` can be automated using the [**Gardener API**](https://github.com/gardener/gardener/blob/master/docs/api-reference/README.md) and a **Service Account** authorized to perform them.
+
+> Note: A project **Service Account** has access to all Kubernetes resources in the project. Treat with care.
```
Comment (petersutter) [inline docs/usage/gardener-api.md:11]: it depends on the given permission ;)

File: docs/usage/working-with-projects.md [hunk 3]
```diff
@@ -1,31 +1,41 @@
...
-    <img class="gs-image" src="../images/00-create-project.png">
+### Steps
 
-2. Provide a project **Name**, and optionally a **Description**, and a **Purpose**, and choose **CREATE**.
+1. Logon to the Gardener Dashboard and choose **CREATE YOUR FIRST PROJECT**.
+  
+    <img class="gs-image" src="../images/00-create-project.png">
 
-    **Note:** You will **not** be able to change the project **Name** later. The rest of the details are editable.
+2. Provide a project **Name**, and optionally a **Description**, **Cost Center** and a **Purpose**, and choose **CREATE**.
```
Comment (petersutter) [inline docs/usage/working-with-projects.md:19]:
```suggestion
2. Provide a project **Name**, and optionally a **Description** and a **Purpose**, and choose **CREATE**.
```
also remove the cost center part from the screenshot below

Comment (n-boshnakov) [reply docs/usage/working-with-projects.md:19]: In order to provide a screenshot without the Cost Center part, I would need access to a Gardener Dashboard that lacks the Cost Center option in order to take an undoctored screenshot, or such a screenshot provided to me.

File: docs/usage/gardener-api.md (hunk 2 above)
Comment (n-boshnakov) [reply docs/usage/gardener-api.md:11]: Would the following correction work? 

"Note: Depending on the given permission, a project **Service Account** could have access to all Kubernetes resources in the project. Treat with care."

File: docs/usage/working-with-projects.md (hunk 3 above)
Comment (n-boshnakov) [reply docs/usage/working-with-projects.md:19]: An option to proceed without a new screenshot would be to simply blur out the Cost Center requirement. Would that work for you?

Comment (petersutter) [reply docs/usage/working-with-projects.md:19]: yes  👍

File: docs/usage/gardener-api.md (hunk 2 above)
Comment (petersutter) [reply docs/usage/gardener-api.md:11]: I would even remove the complete note. Not sure how much value it adds. 

Instead the downloaded kubeconfig (described at the end of the document) should be treated with care as it contains credentials. Maybe this is a more meaningful hint

**Review by petersutter — APPROVED** [review]
/lgtm

**Review by holgerkoser — APPROVED** [review]
/lgtm

**Comment by n-boshnakov** [comment]
Due to rebase issues, I will be closing this PR and opening a new one - #1336

**Pull request closed without merging**